_posts/2018-08-24-running-strimzi-on-aks.md

```diff
@@ -0,0 +1,185 @@
+  "kubernetesVersion": "1.9.9",
+  "linuxProfile": {
+    "adminUsername": "azureuser",
+    "ssh": {
+      "publicKeys": [
+        {
+          "keyData": "ssh-rsa <key-data>"
+        }
+      ]
+    }
+  },
+  "location": "northeurope",
+  "name": "strimzicluster",
+  "networkProfile": {
+    "dnsServiceIp": "10.0.0.10",
+    "dockerBridgeCidr": "172.17.0.1/16",
+    "networkPlugin": "kubenet",
+    "networkPolicy": null,
+    "podCidr": "10.244.0.0/16",
+    "serviceCidr": "10.0.0.0/16"
+  },
+  "nodeResourceGroup": "MC_strimzigroup_strimzicluster_northeurope",
+  "provisioningState": "Succeeded",
+  "resourceGroup": "strimzigroup",
+  "servicePrincipalProfile": {
+    "clientId": "f96f2c4a-02b2-49ac-bcb1-f94c9679abc5",
+    "secret": null
+  },
+  "tags": null,
+  "type": "Microsoft.ContainerService/ManagedClusters"
+}
+
+```
+
+> if you want a cluster running a specific Kubernetes version, you can use the `--kubernetes-version` option. All the supported options for the creation command are available at the official [documentation](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create).
+
+# Connect to the cluster
+
+Because you are going to interact with a Kubernetes cluster, you need the `kubectl` tool for doing that.
+In order to install it, you can follow the official Kubernetes documentation [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/) or using the handy `az aks install-cli` that will do it for you.
+
+After downloading and installing the `kubectl` tool, you need to configure it with the right credentials for connecting to the just deployed Kubernetes cluster in AKS and the following command will do it.
+
+```
+az aks get-credentials --resource-group strimzigroup --name strimzicluster
+```
+
+To verify that the connection with the Kubernetes cluster is working you can just run the `kubectl get nodes` command for showing all the available nodes in the cluster.
+
+```
+NAME                       STATUS    ROLES     AGE       VERSION
+aks-nodepool1-24085136-0   Ready     agent     7m        v1.9.9
+aks-nodepool1-24085136-1   Ready     agent     7m        v1.9.9
+aks-nodepool1-24085136-2   Ready     agent     7m        v1.9.9
+```
+
+# Deploying Strimzi and an Apache Kafka
+
+Now that we have a Kubernetes cluster up and running in the cloud, we can deploy Strimzi in order to deploy an Apache Kafka cluster through the Cluster Operator.
+First of all you have to get the latest Strimzi release from [here](https://github.com/strimzi/strimzi-kafka-operator/releases).
+The first step is to modify the installation files accordingly to the namespace the Cluster Operator is going to be installed.
+
+```
+sed -i 's/namespace: .*/namespace: <my-namespace>/' examples/install/cluster-operator/*ClusterRoleBinding*.yaml
+```
+
+> If you want to deploy the Cluster Operator in the Kubernetes `default` namespace you have just to use `default` for the `<my-namespace>` placeholder in the above command.
```

Comment:
I would be more assertive. Explain, but then say "We're going to install the operator in the default namespace" and give the example as 
```
sed -i 's/namespace: .*/namespace: default/' examples/install/cluster-operator/*ClusterRoleBinding*.yaml
```

Why? Because for people following along it's easier to copy and paste a working example and to have to figure out the placeholders.